pyproject.toml

```diff
@@ -0,0 +1,96 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "tell"
+description = "An open-source Python package to model future hourly total electricity loads."
+readme = "README.md"
+version = "1.2.1"
+requires-python = ">=3.9,<3.11"
```

Comment:
This is stating <3.11 but in the version specifiers below you state you can use 3.11.  I think this should be changed to ">=3.9,<3.12"

Reply:
I was getting errors with 3.11, are you able to install it with 3.11?

Reply:
Let me test in just a few mins.  I'll post back here and let you know.

Reply:
Ok, the error on 3.11 is due to an issue in the compilation of Cython files in scikit-learn while attempting to install the package.  So let's place a ceiling of 3.10 as you recommend, and adjust the available version in the classifiers in the pyproject file, and we will deal with this issue in another PR.

Reply:
sounds good